src/web/routes/application/select-address/select-address.js

```diff
@@ -37,11 +39,14 @@ const findAddress = (selectedAddress, postcodeLookupResults) => {
 }
 
 const behaviourForPost = () => (req, res, next) => {
-  const address = findAddress(req.body.selectedAddress, req.session.postcodeLookupResults)
-  req.session.claim = {
-    ...req.session.claim,
-    ...transformAddress(address)
+  if (req.body.selectedAddress) {
```

Comment:
Bugfix, only transform the address if one has been selected. Can move to own pr if needed.